lib/server/push.api.js

```diff
@@ -24,6 +24,14 @@ Push.setBadge = function (/* id, count */) {
   // throw new Error('Push.setBadge not implemented on the server');
 };
 
+Push.getBadgeNumber = function() {
+  return 1;
+};
+
+Push.setBadgeFunction = function(func) {
```

Comment:
@IlicPavle Remove this method. We will use directly `Push.getBadgeNumber = function() {}` where we need to override badge count.